session.rst

```diff
@@ -3,6 +3,11 @@ Sessions
 
 .. toctree::
     :maxdepth: 1
-    :glob:
 
-    session/*
+    session/sessions_directory
+    session/avoid_session_start
+    session/locale_sticky_session
+    session/limit_metadata_writes
+    session/php_bridge
+    session/proxy_examples
```

Comment:
@javiereguiluz could we also link to `doctrine/pdo_session_storage` in here without creating issues? I actually think it should be the first or second article.